docfiles/script.html

```diff
@@ -22,6 +22,7 @@ <h2 class="ui item header">@title@</h2>
 					</div>
 					<div class="page-header-item show-code-toggle">
 						<button id="show-code-button" class="ui tiny default button">Show Code</button>
+						<a href="/@versionsuff@#pub:@id@" class="ui tiny button landscape only">Edit Code</a>
```

Comment:
This is not a button?

Reply:
A link that works as a button (does not get blocked by pop up blockers).